test/api_integration/apis/index_patterns/fields_for_wildcard_route/conflicts.js

```diff
@@ -14,31 +14,54 @@ export default function ({ getService }) {
         .query({ pattern: 'logs-*' })
         .expect(200)
         .then(resp => {
-          expect(resp.body).to.eql({
-            fields: [
-              {
-                name: '@timestamp',
-                type: 'date',
-                aggregatable: true,
-                searchable: true,
-                readFromDocValues: true,
-              },
-              {
-                name: 'success',
-                type: 'conflict',
-                aggregatable: false,
-                searchable: false,
-                readFromDocValues: false,
-                conflictDescriptions: {
-                  boolean: [
-                    'logs-2017.01.02'
-                  ],
-                  keyword: [
-                    'logs-2017.01.01'
-                  ]
-                }
-              }
-            ]
+          const dateField = resp.body.fields.find(f => f.type === 'date');
+          const successField = resp.body.fields.find(f => f.type === 'conflict');
+          expect(dateField).to.eql({
```

Comment:
These tests intentionally test the entire response body. If it changes at all these tests should fail. Mind not filtering but asserting that the entire body matches?